lib/compress/zstd_compress.c

```diff
@@ -1579,7 +1584,18 @@ ZSTD_reset_matchState(ZSTD_matchState_t* ms,
 
     ZSTD_cwksp_clear_tables(ws);
 
-    DEBUGLOG(5, "reserving table space");
+    /* opt parser space */
```

Comment:
Why is this moved above tables?

Reply:
According to `zstd_cwksp.h` we're actually supposed to allocate all our aligneds before tables. That hadn't been enforced before (tables and aligneds had the same phase), but now that we do distinct stuff before the tables, we have to enforce the ordering.